### DISCUSSION TITLE
skywalking+opentelemetry,How to configure export Trace Data?

### DISCUSSION THREAD

**Discussion by zhangjiabin1010** [apache/skywalking#7970] [Q&A] [answered]
I try to collect trace data using OpentElemetry'S OLTP and export the trace data to SkyWalking's OAP Server. My otel-collector config is as follows:

`receivers:
  otlp:
    protocols:
      grpc:
      http:

exporters:
  logging:
    loglevel: debug

  opencensus:
    endpoint: 127.0.0.1:11800
    insecure: true
    
  jaeger:
    endpoint: 127.0.0.1:14250
    insecure: true  
    

processors:
  batch:
service:
  pipelines:
    traces:
      receivers: [otlp]
      exporters: [jaeger, logging, opencensus]
      processors: [batch]`
      
 But there is an error in my otel-collector：
Sender failed {"component_kind": "processor", "component_type": "batch", "component_name": "batch", "error": "EOF".
Get some help. Thank you

**Comment by zhangjiabin1010** [comment]
In addition, it is normal for me to use SkyWalking's agent to collect data and send it to OAP Server.

**Comment by wankai123** [comment, marked as answer]
I think Skywalking `OpenTelemetry receiver` is used to receive metrics, not traces. 
see https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-receivers/#opentelemetry-receiver

**Comment by yujianweilai** [comment]
Hi ，body:

My situation:
Previously, we used skywalking-agent to transport trace data to skywalking-server. Now I continue to used the skywalking-agent and send the tracing data to otel-collector
, and then otel-collector forwards it to jaeger for storage and display.
The problem is that the tracing data cannot be passed to otel-collector.

My configuration:
/*otel-collector*/
![image](https://github.com/user-attachments/assets/602a6589-42c9-47d0-9524-d8c0e3f49065)

/*skywalking-agent config*/
[root@cx-gateway-service-4100-6b6bdd45cc-pvj98 /]# env|grep SW
SW_LOGGING_DIR=/var/log/cx-gateway-service
SW_AGENT_NAME=cx-gateway-service
SW_AGENT_COLLECTOR_BACKEND_SERVICES=10.100.220.63:11800 # IP 10.100.220.63 is one of otel-collector node address
SW_LOGGING_MAX_HISTORY_FILES=50

Software Version:
skywalking-agent： 8.0.1
otel-collector： 0.85.0

According to the official documentation, when OTEL interconnects with Skywalking, there are few configuration steps. I don't know what steps I'm missing ？ Expect help from experts ，Thanks

**Reply by wu-sheng** [reply]
Simple answer, we don't support that.

**Reply by wu-sheng** [reply]
Our agents require our backend. You can't do things. We did more complex things compared with otel tracing.
